<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>519</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Студенты</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Введите имя первого файла</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QLineEdit" name="file1_line">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>file1.bin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Введите имя второго файла</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QLineEdit" name="file2_line">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>file2.bin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>275</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>Действия с файлами:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view_1">
          <property name="text">
           <string>Просмотреть список 1 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view_2">
          <property name="text">
           <string>Просмотреть список 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="intersection_but">
          <property name="text">
           <string>Найти пересечение списков</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="union_but">
          <property name="text">
           <string>Найти объединение списков</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dif1_2">
          <property name="text">
           <string>Найти разность 1-2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dif2_1">
          <property name="text">
           <string>Найти разность 2-1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>492</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Добавление/удаление:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="number_line">
            <property name="placeholderText">
             <string>Номер зачётки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_line">
            <property name="placeholderText">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="group_line">
            <property name="placeholderText">
             <string>Номер группы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="average_line">
            <property name="placeholderText">
             <string>Средний балл</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="add_1_button">
          <property name="text">
           <string>Добавить в файл 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletr_1_button">
          <property name="text">
           <string>Удалить из файла 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_2_button">
          <property name="text">
           <string>Добавить в файл 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_1_button">
          <property name="text">
           <string>Удалить из файла 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2_1">
   <property name="text">
    <string>Разность 2-1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
